jplag.frontend-utils/src/main/java/de/jplag/Language.java

```diff
@@ -33,7 +34,7 @@ public interface Language {
      * @param files are the names of the files to parse.
      * @return the list of parsed JPlag tokens.
      */
-    TokenList parse(File directory, String[] files);
+    List<Token> parse(File directory, String[] files);
```

Comment:
Maybe we should think about `List<? extends Token>` as this would make it clear that the list is only readable.

Reply:
Resolved by improving the documentation